common/txmgr/address_state.go

```diff
@@ -0,0 +1,240 @@
+package txmgr
+
+import (
+	"sync"
+	"time"
+
+	"gopkg.in/guregu/null.v4"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	feetypes "github.com/smartcontractkit/chainlink/v2/common/fee/types"
+	txmgrtypes "github.com/smartcontractkit/chainlink/v2/common/txmgr/types"
+	"github.com/smartcontractkit/chainlink/v2/common/types"
+)
+
+// AddressState is the state of all transactions for a given address
+type AddressState[
```

Comment:
Plz unexport all types that don't need to be used outside of the txmgr package. 
Comment applies to all other files, types and methods like "NewAddressState"